### DISCUSSION TITLE
只能挂在一个存储吗？

### DISCUSSION THREAD

**Discussion by sucooer** [spencerwooo/onedrive-vercel-index#861] [Q&A]
如题，能否挂载多个存储？

**Comment by spencerwooo** [comment]
暂时不能，可以考虑部署多个站点来分别挂多个 OneDrive 存储。

**Reply by yiranxiamo** [reply]
> 暂时不能，可以考虑部署多个站点来分别挂多个 OneDrive 存储。

1，挂载多个存储，增加阿里云盘，天翼云盘，123盘，夸克
2，暗黑模式，多主题也行
3，视频播放器能否支持同级目录的同名字幕文件呢？
4，能有管理后台就好了。
5，下载次数限制，分享链接下载按次，按时效
考虑下吧
